lib/ui/window/platform_message_response_dart.cc

```diff
@@ -19,30 +19,40 @@ namespace {
 // Avoid copying the contents of messages beyond a certain size.
 const int kMessageCopyThreshold = 1000;
 
+class DataWrapper {
```

Comment:
I think this `DataWrapper` would be unnecessary if we just use `ToByteData`'s implementation inside `WrapByteData`:

```
Dart_Handle WrapByteData(std::unique_ptr<fml::Mapping> mapping) {
  Dart_Handle data_handle =
      Dart_NewTypedData(Dart_TypedData_kByteData, mapping->GetSize());
  if (Dart_IsError(data_handle))
    return data_handle;

  Dart_TypedData_Type type;
  void* data = nullptr;
  intptr_t num_bytes = 0;
  FML_CHECK(!Dart_IsError(
      Dart_TypedDataAcquireData(data_handle, &type, &data, &num_bytes)));

  memcpy(data, buffer.data(), num_bytes);
  Dart_TypedDataReleaseData(data_handle);
  return data_handle;
}
```

Reply:
I intend to optimze according to orignal code. The origial code if if (data.size() < kMessageCopyThreshold) , it will use dart vm heap to allocate data, else it use application heap to alloate data, dart vm only use DataWrapper.
If use ToByteData will consume more dart vm heap.